src/sql/workbench/browser/modelComponents/dropdown.component.ts

```diff
@@ -35,6 +38,16 @@ import { ILogService } from 'vs/platform/log/common/log';
 		</div>
 		<div [style.display]="getEditableDisplay()" #editableDropDown style="width: 100%;"></div>
 		<div [style.display]="getNotEditableDisplay()" #dropDown style="width: 100%;"></div>
+		<label #errorMessage tabindex="-1"
```

Comment:
Why the tabindex of -1? This doesn't seem like something that would be focusable regardless.

Reply:
Without it the screen reader was not reading the error message.